app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -153,6 +153,17 @@ register_katello_host(){
 # Set up subscription-manager
 <%= snippet("subscription_manager_setup", variables: { subman_setup_scenario: 'registration' }).strip -%>
 
+<% unless @repo_data.blank? -%>
+# clean repositories which were added for registration purposes
```

Comment:
If I understand correctly, it's for cases where the host is registered with `subscription-manager`. So, after the successful registration, we want to consume content provided by the subman configuration, right?

So, how about the use case where users don't use a subscription manager (no Katello)?
They would like to continue consuming such content, right?

Reply:
I guess so. And this is exactly what happens in the code. If you look at line 133, then everything that happens until line 171 is in Katello context - including the cleanup of the repositories.